### ISSUE TITLE
RuntimeException: Unable to pause activity {org.wordpress.android/org.wordpress.android.ui.prefs.BlogPreferencesAct...

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-Android#15409]
**Sentry Url**: https://sentry.io/share/issue/1def90e49e0847b89336cd48a5155c0d/
**User Count**: 655
**Count**: 1607
**First Release**: org.wordpress.android-12.9
**First Seen**: 2019-08-10T08:09:28Z
**Last Seen**: 2021-10-04T08:09:05Z
**24 Hours**: 47
**30 Days**: 661

```
RuntimeException: Unable to pause activity {org.wordpress.android/org.wordpress.android.ui.prefs.BlogPreferencesActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.wordpress.android.fluxc.model.SiteModel.setName(java.lang.String)' on a null object reference
     at org.wordpress.android.ui.prefs.SiteSettingsFragment.updateTitle(SiteSettingsFragment.java:889)
     at org.wordpress.android.ui.prefs.SiteSettingsFragment.onPause(SiteSettingsFragment.java:371)
...
(14 additional frame(s) were not displayed)
```

This one seems quite old at first… but suddenly spiking recently since Sept 22, 2021 and 18.2 😱
![image](https://user-images.githubusercontent.com/216089/135828076-92cd9a96-3d83-42e5-8e55-54bc8cca9b47.png)

**Comment by dcalhoun** [comment]
Labeling this as `[Pri] Medium` after discussing with @renanferrari. This issue appears to be present for older releases, but a significant spike occurred around the 18.2 release on 22 Sept. 672 users impacted, which I estimate to be 0.36% for the release. 

From reviewing the breadcrumbs, it would appear something in the Blogging Reminders fragment triggers `SiteModel.setName`, but on a `null` model. I did notice site deletion requests and unstable network connectivity events in the breadcrumbs, which may result in the target site not being available.

**Comment by renanferrari** [comment]
After taking a look at this, it seems this wasn't related to Blogging Reminders after all. Instead, it's related to a bug during site deletion, where the site model can be null:

https://user-images.githubusercontent.com/830056/140549154-790af8be-f129-4236-9a60-4c67168ce373.mp4

I submitted a fix for it here: #15536
